install-manager/src/main/java/cm/aptoide/pt/install_manager/dto/InstallPackageInfo.kt

```diff
@@ -7,6 +7,7 @@ data class InstallPackageInfo(
   val versionCode: Long,
   val downloadSize: Long = Long.MIN_VALUE,
   val installationFiles: Set<InstallationFile> = emptySet(),
-  val payload: String? = null
+  val payload: String? = null,
+  val isAppCoins: Boolean = false
```

Comment:
Let's not add it here.
Install manager should not be aware of such kind of details.
Pack it up into `payload`.